react/Section.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+import { useResponsiveValue } from 'vtex.responsive-values'
+
+interface Props {
+  borderless?: boolean
+  marginBottom?: number
+  paddingBottom?: number
+  width?: string | { mobile: string; desktop: string }
+  name: string
+}
+
+const CSS_HANDLES = ['section']
+
+const Section: FC<Props> = ({
+  children,
+  borderless = false,
+  marginBottom,
+  paddingBottom,
+  width,
+  name,
+}) => {
+  const handles = useCssHandles(CSS_HANDLES)
+  const responsiveWidth = useResponsiveValue(width)
```

Comment:
Just an opinion. You could call the width prop of `width: widthProp` and then use it in the rest of the component as just `width` instead of responsiveWidth

```typescript
const width = useResponsiveValue(width)
```

Reply:
💅